app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainFragment.kt

```diff
@@ -94,8 +94,12 @@ class MainFragment : Fragment() {
 
     fun getClickedStock(view: View) {
         Log.d("[SEARCH BAR]", "Clicked a stock.")
+
     }
 
+    fun createStockList(view: View) {
+
+    }
```

Comment:
So I started this function, maybe you guys can finish it because im not sure. But this would allow the user to have a watchlist of their favorite stocks and ones they are watching.

Reply:
This is an incomplete feature.

1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.